README.md

```diff
@@ -196,17 +196,23 @@ repeating the mistakes of the past.
       if (isRunningInApp)
       ```
 
-#### Write [Minimum Viable Documentation](https://mobify.atlassian.net/wiki/pages/viewpage.action?pageId=50266690#OurSoftwareDesign&DevelopmentProcess-MinimumViableDocumentation)
-- All modules/libraries/plugins/etc. should have documentation that explains:
-    - Setup
-    - Usage
-    - Deployment
-    - Testing
-    - Contributing
-    - Roadmap/Changelog
-- Good documentation enables others to get up and running without your involvement.
-- Documentation should be updated as you write your software (or [even before you write it](http://tom.preston-werner.com/2010/08/23/readme-driven-development.html)))
-- You should be able to answer the majority of questions from users with a link to a document.
+#### ✏️ Write Documentation
+
+Documentation is essential to a codebase accessible for new contributors.
+
+All projects, modules, libraries and plugins must include a `README.md` file in
+the root folder. Minimum viable documentation includes:
+
+* 🤔 The **purpose** of the project.
+* 👋 Who the **owner** of the project is, and **where to ask for help**.
+* 🏗 Steps to **setup** the project. _Preferably this is one command._
+* 🏃 Steps to **run** the project. _Preferably this is one command._
+* 🐛 Steps to **test** the project. _Preferably this is one command and we've hooked up to CI._
+* 🚀 Steps to **deploy** the project. _Preferably this is one command and we've hooked up to CD._
+* 🎖 Steps to **contribute** to the project. _Considering using [`CONTRIBUTING.md`](https://github.com/blog/1184-contributing-guidelines)._
+
+[Draft your documentation before committing](http://tom.preston-werner.com/2010/08/23/readme-driven-development.html)
```

Comment:
Definitely want us to link to this article - a bit confused why the think is called "Draft your documentation before commiting" though. Should this instead be "Draft your documentation before committing *any code*"?